vendor_tests/_array_api_compat_vendor.py

```diff
@@ -0,0 +1,9 @@
+# This file is a hook imported by src/array_api_extra/_lib/_compat.py
+from __future__ import annotations
+
+from .array_api_compat import *  # noqa: F403
+from .array_api_compat import array_namespace as array_namespace_compat
```

Comment:
why do we want to expose this as `array_namespace_compat` too?

Reply:
Because we're overriding it immediately below. We're overriding it so that we can safely test `is` in the unit test. Adding a comment.